app/src/main/res/layout/fragment_send_messages.xml

```diff
@@ -0,0 +1,52 @@
+<layout xmlns:android="http://schemas.android.com/apk/res/android">
+
+    <data>
+
+        <variable
+            name="viewModel"
+            type="com.teamagam.gimelgimel.app.message.viewModel.SendMessagesViewModel" />
+    </data>
+
+    <LinearLayout
+        android:id="@+id/fragment_messages_text"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="16dp"
+        android:orientation="horizontal">
+
+        <LinearLayout
+            android:layout_width="0dp"
+            android:layout_height="@dimen/fab_normal"
+            android:layout_weight="1"
+            android:background="@drawable/message_balloon"
+            android:orientation="horizontal">
+
+            <EditText
+                android:id="@+id/send_text_message_view"
+                android:layout_width="0dp"
+                android:layout_height="@dimen/fab_normal"
+                android:layout_weight="1"
+                android:background="@android:color/transparent"
+                android:hint="@string/text_message_hint"
+                android:text="@={viewModel.text}" />
+
+            <ImageButton
+                android:id="@+id/send_image_button"
+                android:layout_width="32dp"
+                android:layout_height="32dp"
+                android:layout_gravity="center"
+                android:background="@drawable/ic_camera"
+                android:backgroundTint="@color/gray_dark"
+                android:contentDescription="@string/send_image_fab_desc"
+                android:onClick="@{() -> viewModel.onSendImageButtonClicked()}" />
+        </LinearLayout>
+
+        <android.support.design.widget.FloatingActionButton
+            android:id="@+id/send_text_message_fab"
+            style="@style/fab"
```

Comment:
override style/fab margin's to allow button be centered.
use layout_margin="0dp"
and
layout_gravity="center"